src/main/java/org/tc/controllers/CourseController.java

```diff
@@ -255,4 +288,71 @@ public ModelAndView myCourses(@ModelAttribute("category") Category category) {
         mav.addObject(COURSES_OBJECT_NAME, courseDTOConverter.convertAll(myCourses));
         return mav;
     }
+
+    @RequestMapping(value = "/send_to_review/{id}", method = RequestMethod.GET)
+    public ModelAndView sendToReview(@PathVariable("id") int id) {
+        Course course = courseService.getById(id);
+        if (!courseService.isProposal(course) && courseService.isOwner(course)) {
```

Comment:
Course state is, it's only methods where I check for equals with certain state: isProposal, isDraft, isRejected